[build-system]
requires = [
  "setuptools>=61.0.0",
]
build-backend = "setuptools.build_meta"

[project]
name = "pytest-errxfail"
description = "pytest plugin to mark a test as xfailed if it fails with the specified error message in the captured output"
version = "0.1.0"
readme = "README.rst"
requires-python = ">=3.8"
authors = [
    { name = "eltio", email = "eltio.sw@mail.ru" },
]
maintainers = [
    { name = "eltio", email = "eltio.sw@mail.ru" },
]
license = {file = "LICENSE"}
classifiers = [
    "Framework :: Pytest",
    "Development Status :: 4 - Beta",
    "Intended Audience :: Developers",
    "Topic :: Software Development :: Testing",
    "Operating System :: OS Independent",
    "Programming Language :: Python",
    "Programming Language :: Python :: 3.8",
    "Programming Language :: Python :: 3.9",
    "Programming Language :: Python :: 3.10",
    "Programming Language :: Python :: 3.11",
    "Programming Language :: Python :: 3.12",
    "Programming Language :: Python :: 3 :: Only",
    "Programming Language :: Python :: Implementation :: CPython",
    "Programming Language :: Python :: Implementation :: PyPy",
    "License :: OSI Approved :: MIT License",
]
dependencies = [
    "pytest>=6.2.0",
]
[project.urls]
Repository = "https://github.com/eltimen/pytest-errxfail"
[project.entry-points.pytest11]
errxfail = "pytest_errxfail.errxfail"
